packages/components/src/date-time/style.scss

```diff
@@ -1,126 +1,217 @@
-$datepicker__border-radius: 0;
-$datepicker__day-margin: 0.166rem;
-$datepicker__font-size: 13px;
-$datepicker__font-family: inherit;
-$datepicker__item-size: 28px;
-$datepicker__margin: 0.4rem;
-$datepicker__navigation-size: 6px;
-$datepicker__triangle-size: 6px;
-
-$datepicker__background-color: $light-gray-300;
-$datepicker__text-color: $dark-gray-500;
-$datepicker__header-color: $black;
-$datepicker__muted-color: #ccc;
-$datepicker__navigation-disabled-color: lighten($datepicker__muted-color, 10%);
-$datepicker__border-color: $light-gray-500;
-
-// Lerna hoist packages, so can't reference with ~
-@import "../../../../node_modules/react-datepicker/src/stylesheets/datepicker";
-
-.react-datepicker__month-container {
-	float: none;
-}
-
-.react-datepicker-time__header,
-.react-datepicker__current-month {
-	font-size: $datepicker__font-size;
-}
+// We can't reference this package with ~ because of how Lerna handles packages. 😩
+@import "../../node_modules/react-dates/lib/css/_datepicker.css";
+
+.components-datetime-date {
+	min-height: 236px;
+	border-top: 1px solid $light-gray-500;
+	margin-left: -$item-spacing;
+	margin-right: -$item-spacing;
+
+	.components-datetime-date__aria-description {
+		position: absolute;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		background-color: $white;
+		z-index: 100;
+		padding: $item-spacing;
+
+		&--state-invisible {
```

Comment:
For me, the Help is always visible and hides the whole panel. This doesn't seem to work as intended, as the class name actually used is `.is-invisible`

Reply:
Sorry, I pushed a WIP patch and it messed things up. This isn't ready for a review right now. I'll try to get it ready ASAP and will ping you then.